todo_app/templates/index.html

```diff
@@ -10,18 +10,36 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
+
       <ul class="list-group mb-4">
         {% for item in items %}
-        <li>{{ item.title }} -- {{ item.status }}</li>
-        <form id="completeItem{{item.id}}" method="POST" action="/complete/{{item.id}}">
-          <input type="submit" value="Complete Item" />
-        </form>
-        <a onclick="deleteItem({{ item.id }})" href="#">Delete</a>
+          <li>{{ item.name }} -- {{ item.desc }} -- {{ lists[item.idList]['name'] }}</li>
+          
+          {% if item.idList == '607f1ee7c99b117ea947851b' %}
+            <form id="completeItem{{item.id}}" method="POST" action="/complete/{{item.id}}/607f1ee7c99b117ea947851c">
```

Comment:
It's quite hard to parse this logic - rather than matching list ids exactly, it might make more sense to:
* Split your `complete` endpoint into several, e.g. start, complete & restart
  * This would help make the urls much easier to reason about
  * It would also mean that the list ids don't have to be passed through to the template explicitly so we could then:
* Store the list names (or similar) on the Item objects rather than the list ids - so e.g. here we could compare like:
`if item.list_name == 'To Do'`

Reply:
done